articles/spring-cloud/how-to-use-enterprise-spring-cloud-gateway.md

```diff
@@ -164,28 +91,29 @@ Here's a sample of the JSON file that is passed to the `--routes-file` parameter
-| tags        | Classification tags, which will be applied to methods in the generated OpenAPI documentation.                                                                                                                                                                                                                                       |
-
-Not all the filters/predicates are supported in Azure Spring Apps because of security/compatible reasons. The following aren't supported:
+| Property | Description |
+| - | - |
+| title | A title to apply to methods in the generated OpenAPI documentation |
+| description | A description to apply to methods in the generated OpenAPI documentation  |
+| uri | The full URI, which will override the name of app that requests route to. |
+| ssoEnabled | A value that indicates whether to enable SSO validation. See [Using Single Sign-on](./how-to-configure-enterprise-spring-cloud-gateway.md#configure-single-sign-on-sso) |
+| tokenRelay | Passes the currently authenticated user's identity token to the application |
+| predicates | A list of predicates. See [Available Predicates](https://docs.vmware.com/en/VMware-Spring-Cloud-Gateway-for-Kubernetes/1.0/scg-k8s/GUID-configuring-routes.html#available-predicates) |
+| filters | A list of filters. See [Available Filters](https://docs.vmware.com/en/VMware-Spring-Cloud-Gateway-for-Kubernetes/1.0/scg-k8s/GUID-configuring-routes.html#available-filters) |
+| order | The route processing order - a lower order is processed with higher precedence, same as [Spring Cloud Gateway OSS](https://docs.spring.io/spring-cloud-gateway/docs/current/reference/html/) |
+| tags | Classification tags that will be applied to methods in the generated OpenAPI documentation |
 
-- BasicAuth
-- JWTKey
+> [!NOTE]
+> Not all the filters/predicates are supported in Azure Spring Apps because of security or compatibility reasons. The following aren't supported:
+>
+> - BasicAuth
+> - JWTKey
 
-## Create an example application
+## Using Routes for Spring Cloud Gateway for Kubernetes Example
 
-Use the following steps to create an example application using Spring Cloud Gateway for Tanzu.
+Use the following steps to create an example application using Spring Cloud Gateway for Kubernetes.
 
-1. To create an app in Azure Spring Apps that the Spring Cloud Gateway for Tanzu would route traffic to, follow the instructions in [Quickstart: Build and deploy apps to Azure Spring Apps using the Enterprise tier](quickstart-deploy-apps-enterprise.md). Select `customers-service` for this example.
+1. To create an app in Azure Spring Apps that the Spring Cloud Gateway for Kubernetes would route traffic to, follow the instructions in the [Animal Rescue Sample App](https://github.com/Azure-Samples/animal-rescue) through [Build and Deploy Applications](https://github.com/Azure-Samples/animal-rescue#build-and-deploy-applications).
```

Comment:
asking developers to follow a big sample to understand is complicated. Suggest that we have conceptual doc contents without referencing any samples. Otherwise, we are asking the developers to understand the sample to understand the concepts